pinot-controller/src/main/java/com/linkedin/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -481,26 +487,23 @@ private JSONArray getAllSegmentsMetadataForTable(String tableName, CommonConstan
    * XXX
```

Comment:
I know that you didn't add this but I think we should remove this?

Reply:
Done